vendor/k8s.io/kubernetes/openshift-kube-apiserver/openshiftkubeapiserver/patch.go

```diff
@@ -84,6 +86,12 @@ func NewOpenShiftKubeAPIServerConfigPatch(kubeAPIServerConfig *kubecontrolplanev
 		genericConfig.AddPostStartHookOrDie("openshift.io-startkubeinformers", func(context genericapiserver.PostStartHookContext) error {
 			go kubeInformers.Start(context.StopCh)
 			go openshiftInformers.Start(context.StopCh)
+			err := wait.PollImmediate(1*time.Second, 10*time.Second, func() (bool, error) {
```

Comment:
> I would rather expect this much higher up in the outer plumbing code that sets up the group cache, and therefore delay the readiness of the whole apiserver.

@deads2k what do you think of waiting for the groups cache to sync here, as suggested by @sttts ? I think I would then be able to undo the other commit's changes to the `GroupCache`.

Reply:
Don't we wire the WaitForSync funcs somewhere already?